### ISSUE TITLE
Feature Request - Permalink for saved queries

### ISSUE THREAD

**Issue by Noloxs** [humio/issues#67]
Being able to get a permanent link for a saved query would be very useful for cases like documentation.
The link should simply redirect the user to the latest version of the selected query.

Adding a copy permalink button to the saved query would also be helpful

**Comment by mwl** [comment]
You can perform a search for `$MySavedQuery` which will generate a unique url for your search, including time window etc.

**Comment by Noloxs** [comment]
Super. That definitely works for getting the query itself, but the time window that is set for the saved query is lost, which could be an issue for curtain queries and live queries.

So if the time window if unspecified would use the time window defined in the saved query instead of defaulting to 24 hour static, then it would be perfect.

**Comment by mwl** [comment]
Good point.

@anagrius What are your thoughts?

**Comment by anagrius** [comment]
Good point, indeed. It would mean that the search view would have to load the query in question by id,
which it does not do today. Alternatively we make a separate "Permalink" that loads the query and redirects to the search view. Both have pros and cons. Let me think about it.

**Comment by mwl** [comment]
@Noloxs In the meantime what you're trying to achieve can more or less be solved by dashboards.
